src/Sylius/Bundle/ApiBundle/Helpers/AbstractDataPricesTransformer.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace Sylius\Bundle\ApiBundle\Helpers;
+
+use Sylius\Bundle\MoneyBundle\Form\DataTransformer\SyliusMoneyTransformer;
+
+abstract class AbstractDataPricesTransformer
+{
+    protected bool $isOutput;
+    protected SyliusMoneyTransformer $transformer;
+
+    public function __construct()
```

Comment:
When I was said about inject as a service I mean something like that
public function __construct(SyliusMoneyTransformer $syliusMoneyTransformer)
```suggestion
    public function __construct(SyliusMoneyTransformer $syliusMoneyTransformer)
    {
        $this->syliusMoneyTransformer = $syliusMoneyTransformer;
    }
```
But you need ensure that it is registered as a service

Reply:
I didn't see requested changes

Reply:
i use another class to instantiate the pricetransformer (kinda static factory method) so I don't use that one as a service neither